config/install/migrate_plus.migration.node.yml

```diff
@@ -57,6 +58,16 @@ process:
     bundle_key: vid
     bundle: islandora_models 
 
+  # Also populate the "Resrouce type" field from string
+  # values in the 'resource_type' column in the CSV file.
+  field_resource_type:
+    plugin: entity_generate
```

Comment:
For the sake of simplicity, it may be better to just do the same as `field_model` except use the `resource_types` vocab.  The 'Image' term from that vocabulary should populate the field.

Reply:
Well, except maybe make a source constant like you did in https://github.com/Islandora-CLAW/migrate_islandora_csv/pull/15.

I think a source of `constants/resource_type` is much less confusing that `constants/model` if the field name is `field_resource_type`.